------------------------------
:::: AGGREGATED TEST RESULT ::::

============================================================
HNS-OAuth
============================================================

[ERROR] testBackoffRetryMetrics(org.apache.hadoop.fs.azurebfs.services.TestAbfsRestOperation)  Time elapsed: 3.604 s  <<< ERROR!
[ERROR] testReadFooterMetrics(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.503 s  <<< ERROR!
[ERROR] testMetricWithIdlePeriod(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.371 s  <<< ERROR!
[ERROR] testReadFooterMetricsWithParquetAndNonParquet(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.372 s  <<< ERROR!

[ERROR] Tests run: 143, Failures: 0, Errors: 1, Skipped: 2
[ERROR] Tests run: 626, Failures: 0, Errors: 3, Skipped: 76
[WARNING] Tests run: 414, Failures: 0, Errors: 0, Skipped: 57

============================================================
HNS-SharedKey
============================================================

[ERROR] testBackoffRetryMetrics(org.apache.hadoop.fs.azurebfs.services.TestAbfsRestOperation)  Time elapsed: 3.124 s  <<< ERROR!
[ERROR] testReadFooterMetrics(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.026 s  <<< ERROR!
[ERROR] testMetricWithIdlePeriod(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 0.905 s  <<< ERROR!
[ERROR] testReadFooterMetricsWithParquetAndNonParquet(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.012 s  <<< ERROR!

[ERROR] Tests run: 143, Failures: 0, Errors: 1, Skipped: 3
[ERROR] Tests run: 626, Failures: 0, Errors: 3, Skipped: 28
[WARNING] Tests run: 414, Failures: 0, Errors: 0, Skipped: 44

============================================================
NonHNS-SharedKey
============================================================

[ERROR] testBackoffRetryMetrics(org.apache.hadoop.fs.azurebfs.services.TestAbfsRestOperation)  Time elapsed: 3.484 s  <<< ERROR!
[ERROR] testReadFooterMetrics(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.076 s  <<< ERROR!
[ERROR] testMetricWithIdlePeriod(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.079 s  <<< ERROR!
[ERROR] testReadFooterMetricsWithParquetAndNonParquet(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.028 s  <<< ERROR!

[ERROR] Tests run: 143, Failures: 0, Errors: 1, Skipped: 9
[ERROR] Tests run: 610, Failures: 0, Errors: 3, Skipped: 268
[WARNING] Tests run: 414, Failures: 0, Errors: 0, Skipped: 47

============================================================
AppendBlob-HNS-OAuth
============================================================

[ERROR] testBackoffRetryMetrics(org.apache.hadoop.fs.azurebfs.services.TestAbfsRestOperation)  Time elapsed: 3.495 s  <<< ERROR!
[ERROR] testReadFooterMetrics(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.195 s  <<< ERROR!
[ERROR] testMetricWithIdlePeriod(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.239 s  <<< ERROR!
[ERROR] testReadFooterMetricsWithParquetAndNonParquet(org.apache.hadoop.fs.azurebfs.ITestAbfsReadFooterMetrics)  Time elapsed: 1.205 s  <<< ERROR!

[ERROR] Tests run: 143, Failures: 0, Errors: 1, Skipped: 2
[ERROR] Tests run: 626, Failures: 0, Errors: 3, Skipped: 78
[WARNING] Tests run: 414, Failures: 0, Errors: 0, Skipped: 81

Time taken: 55 mins 19 secs.

--- a/hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/TestAccountConfiguration.java
+++ b/hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/TestAccountConfiguration.java
@@ -460,7 +460,7 @@ public class TestAccountConfiguration {
       // Test that we managed to instantiate an MsiTokenProvider without having to define the tenant and client ID.
       // Those 2 fields are optional as they can automatically be determined by the Azure Metadata service when
       // running on an Azure VM.
-      Assertions.assertThat(tokenProviderTypeName).isInstanceOf(MsiTokenProvider.class);
+      Assertions.assertThat(tokenProviderTypeName).describedAs("Token Provider Should be MsiTokenProvider").isInstanceOf(MsiTokenProvider.class);
   }
 
   public void testGlobalAndAccountOAuthPrecedence(AbfsConfiguration abfsConf,

